### ISSUE TITLE
js-debugger no longer functioning

### ISSUE THREAD

**Issue by khan-de-code** [microsoft/vscode-js-debug#578]
**Problem**
The debugger appears to no longer be connecting to nodejs as my app previously would display output in the 'DEBUG CONSOLE' as part of the startup process and would also have no problem previously setting breakpoints. After updating to 1.47.0 the debugger no longer outputs anything to the 'DEBUG CONSOLE' outside of the following:

```
Debugger listening on ws://127.0.0.1:53851/7dae0612-5d84-4222-b06c-7693db629722
For help see https://nodejs.org/en/docs/inspector
Debugger attached.
```

Also, every attempt to set breakpoints now shows them as grey without a center fill. Hovering over them produces the text 'Unbound Breakpoint'.

**To Reproduce**
1. Go to the debug dropdown in the top left and choose the debug configuration associated with the following configuration:
{
      "type": "node",
      "request": "launch",
      "name": "Launch App",
      "protocol": "inspector",
      "program": "${workspaceRoot}/server.js",
      "env": {
 // env vars
      },
      "outFiles": ["${workspaceFolder}/out/**/*.js"]
}

2. Click on the run button.

**Log File**
Unable to add log file due to it including sensitive information.

**VS Code Version:** 1.47.0

**Additional context**
N/A

**Comment by connor4312** [comment]
Unfortunately this alone isn't enough info for me to diagnose the issue, though it sounds similar to https://github.com/microsoft/vscode/issues/102166. If you can make a small standalone reproduction case where you can collect the log file, that would be very helpful.

For the moment you can set `debug.javascript.usePreview: false` to use the old debugger, but please do provide any reproduction information you can as this option will eventually go away.

**Comment by connor4312** [comment]
If you are using Node 8, this may be https://github.com/microsoft/vscode/issues/102166. You can either:

1. Update to Node 10 or higher (Noe 8 is EOL at this point, so it's advisable in general)
1. Set `debug.javascript.usePreview: false` to temporarily use the old debugger
1. Wait for the next [nightly build](https://github.com/microsoft/vscode-js-debug/#nightly-extension) of the extension coming in a few hours and update to that, or wait for the next vscode release early/mid next week

Please let me know if this is the case!

**Comment by connor4312** [comment]
If this is the case, the solution can be found at the bottom of that issue. Let me know if it doesn't work for you and I'll reopen this issue.

Thanks for reporting it!

**Comment by connor4312** [comment]
A fix for this issue has now been released in VS Code 1.47.2. If you previously had to opt-out of the new debugger, you should be safe to turn it back on. Thanks again for the report!
